src/error.rs

```diff
@@ -49,7 +49,11 @@ pub struct FinalError {
 impl Display for FinalError {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
         // Title
-        write!(f, "{}[ERROR]{} {}", *RED, *RESET, self.title)?;
+        if *crate::cli::ACCESSIBLE.get().unwrap_or(&false) {
+            write!(f, "{}ERROR{}: {}", *RED, *RESET, self.title)?;
+        } else {
+            write!(f, "{}[ERROR]{} {}", *RED, *RESET, self.title)?;
+        }
```

Comment:
You changed the title, but forgot to change the hints, I think.

Try running this:

```sh
$ touch input
$ ouch compress input output
```

![image](https://user-images.githubusercontent.com/38900226/142958758-ce730cf1-84b3-4c02-86fc-744340a02f3b.png)

---

Or this:

```sh
$ touch a
$ ouch compress a a.gz.tar
```

![image](https://user-images.githubusercontent.com/38900226/142958511-a1921d53-6592-42f7-8747-8c951b204bf5.png)

Reply:
Good catch!

I've changed to output to this for accessibility mode:
![2021-11-23_11-47-01_Tuesday](https://user-images.githubusercontent.com/12369750/143011135-7a31dae7-68c6-47d7-ba6c-3cb4df618912.png)

Reply:
Perfect!